qualification/requirements.txt

```diff
@@ -203,7 +203,6 @@ numpy==1.23.5
     #   pandas
     #   prometheus-api-client
     #   scipy
-    #   spead2
```

Comment:
Oh dear, it looks like spead2 has a typo in pyproject.toml that makes it not actually depend on numpy. I probably need to release a bugfix release. I think it shouldn't have any bad effects on katgpucbf for now though.